nukebuild/Build.cs

```diff
@@ -414,13 +414,13 @@ void DoMemoryTest()
 
             ApiDiffHelper.MergePackageMarkdownDiffFiles(outputFolderPath, baselineDisplay, currentDisplay);
         });
-    
+
     Target RunTests => _ => _
         .DependsOn(RunCoreLibsTests)
         .DependsOn(RunRenderTests)
         .DependsOn(RunToolsTests)
-        .DependsOn(RunHtmlPreviewerTests)
-        .DependsOn(RunLeakTests);
+        .DependsOn(RunHtmlPreviewerTests);
+        //.DependsOn(RunLeakTests); // dotMemory Unit doesn't support modern .NET versions, see https://youtrack.jetbrains.com/issue/DMU-300/
```

Comment:
Can we have an issue for that? Somewhere in 12.x milestone maybe.

Reply:
Done: https://github.com/AvaloniaUI/Avalonia/issues/20004